test/libs.js

```diff
@@ -12,14 +12,12 @@ const FILES = {
     'scripts/rollup.config.js',
     'scripts/version.js',
     'src/plugin.js',
-    'test/index.html',
     'scripts/karma.conf.js',
     'test/plugin.test.js',
     '.editorconfig',
     '.gitignore',
     '.npmignore',
     '.nvmrc',
-    'scripts/jsdoc.json',
```

Comment:
We only make this file for docs, seems like the tests were majorly broken until now.